grails-app/controllers/au/org/ala/profile/hub/ProfileController.groovy

```diff
@@ -321,6 +321,35 @@ class ProfileController extends BaseController {
         }
     }
 
+    def getPublication() {
+        def pubId = params.pubId;
+        if (!pubId) {
+            badRequest "Publication Id must be provided";
+        } else {
+            def pubJson = profileService.getPublications(pubId)?.resp;
+            if (!pubJson) {
+                notFound()
+            } else {
+                def profile = profileService.getProfile(pubJson.profile.opusId, pubJson.profile.uuid, true);
```

Comment:
Don't use true here. Use this:
            boolean latest = params.isOpusReviewer || params.isOpusEditor || params.isOpusAdmin